<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::ClearHistoryDialog</class>
 <widget class="QDialog" name="Otter::ClearHistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clear History</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="periodWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="periodLabel">
        <property name="text">
         <string>Period to clear:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="periodSpinBox">
        <property name="specialValueText">
         <string>All</string>
        </property>
        <property name="suffix">
         <string> h</string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearBrowsingHistoryCheckBox">
     <property name="text">
      <string>Clear browsing history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearCookiesCheckBox">
     <property name="text">
      <string>Clear cookies</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearFormsHistoryCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Clear forms history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearDownloadsHistoryCheckBox">
     <property name="text">
      <string>Clear downloads history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearSearchHistoryCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Clear search history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearCachesCheckBox">
     <property name="text">
      <string>Clear caches</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearStorageCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Clear websites storage data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearPasswordsCheckBox">
     <property name="text">
      <string>Clear passwords</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Otter::ClearHistoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::ClearHistoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
